### DISCUSSION TITLE
Global metrics

### DISCUSSION THREAD

**Discussion by iamjoostjanssen** [brainvisa/brainvisa.github.io#137] [Q&A]
Hi,

I use a very old version (4.6) and therefore, perhaps, my remark is outdated. if so, then I apologize beforehand. It would be a great idea, IMO, to have global metrics (e.g. mean hemispheric sulcal depth or total hemispheric sulcal length) included in the default-output morphometry calculations. 

Merci,
-joost

**Comment by denisri** [comment]
Hi,
No your remark is not outdated at all since we have never took the time to include such metrics in the pipeline. There are several processes which can provide a few things (brain volumes, surface area and volumes, global sulcal index...) but none was included in the default pipeline, and none was providing total sulci length, or average thickness or opening...
As you are right to ask for it, and I wanted to include it for a long time, it has been the occasion to do it. I have just added one in the development sources...
Denis
